core/domain/interaction_registry_test.py

```diff
@@ -87,6 +88,9 @@ class InteractionRegistryUnitTests(test_utils.GenericTestBase):
 
     def test_interaction_registry(self):
         """Do some sanity checks on the interaction registry."""
+        with patch(
+                'core.domain.interaction_registry.Registry._interactions', {}):
+            interaction_registry.Registry.get_all_interactions()
```

Comment:
Why are you using `patch` here instead of `self.swap` or `self.swap_with_checks` like we usually do in backend tests? I'm not particularly opposed to `patch`--I just want to understand the rationale behind the choice

Reply:
I used patch instead of self.swap mainly because of my familiarity with patch, but being consistent with the rest of the backend tests is much more important. I'll use swap instead.